Source/RenderStreamEditor/Private/RenderStreamEditorModule.cpp

```diff
@@ -589,15 +592,17 @@ void UpdateChannelCache()
 
     // Loop over all levels and make sure caches exist for them.
     TArray<FAssetData> LevelAssets;
-    const auto LevelLibrary = UObjectLibrary::CreateLibrary(ULevel::StaticClass(), false, true);
+    const auto LevelLibrary = UObjectLibrary::CreateLibrary(UWorld::StaticClass(), false, true);
     LevelLibrary->LoadAssetDataFromPath(ContentFolder);
```

Comment:
This is the important part, apparently at some point they changed it so you can no longer load levels directly but you have to load worlds instead (which are technically a collection of levels but the persistant level which we care about is always the first one)

Reply:
Would that work on a 4.27 project?

I know the plugin says 5.1, but we know of setups running 4.27 project on 5.1 plugins.

Reply:
Yes, the main difference between the 4.27 version of this change and 5.x is the FSavePackageArg above doesn't exist.
But this one will work like this (Jordan confirmed the current released version of 4.27 does not work)